gateway/up.go

```diff
@@ -18,6 +19,13 @@ func constructReq(ctx context.Context, req *entity.UpRequest) (*http.Request, er
 		return nil, err
 	}
 	httpReq.Header.Set("Content-Type", "multipart/form-data")
+
+	version := constants.Version
+	if constants.IsDevVersion() {
+		version = "dev"
+	}
+	httpReq.Header.Set("X-Railway-Version", version)
```

Comment:
Do we want to send this header on all requests (including GQL)? If so, we can pull the http client out of cli/gateways/main.go into a common client and configure it to add the header on every request.

Reply:
Yes that is a good idea